package.json

```diff
@@ -35,10 +35,10 @@
   },
   "peerDependencies": {
     "cldr-data": ">=25",
-    "globalize": ">=1.1.0-rc.5"
+    "globalize": "^1.1.0"
   },
   "devDependencies": {
-    "babel": "^5.6.14",
+    "babel": "^6.3.26",
```

Comment:
Actually, we support babel `^5.6` or `6.x`, right? If so, we don't need to restrict it to 6.x only.